src/jit/importer.cpp

```diff
@@ -6921,13 +6921,13 @@ bool  Compiler::impMethodInfo_hasRetBuffArg(CORINFO_METHOD_INFO * methInfo)
     // Support for any additional cases that don't use a Return Buffer Argument
     //  on targets that support multi-reg return valuetypes.
     //
-  #ifdef _TARGET_ARM_
+  #ifdef FEATURE_HFA
     // On ARM HFAs are returned in registers.
     if (!info.compIsVarArgs && IsHfa(methInfo->args.retTypeClass))
     {
         return false;
     }
-  #endif
+  #endif // FEATURE_HFA
```

Comment:
If IsHfa(hClass) is defined as I have suggested earlier, there is no need to enclose this logic inside #ifdef FEATURE_HFA.